### DISCUSSION TITLE
[Bug Report] Cannot run sensors tutorials code

### DISCUSSION THREAD

**Discussion by gonzalezJohnas** [isaac-sim/IsaacLab#1145] [Q&A]
### Describe the bug

I cannot launch any of tutorials 04_sensors, the core dumped appear as soon as I use any sensors ( Camera, RayCasterCfg...). If I comment them everything works fine.

### Steps to reproduce

python source/standalone/tutorials/04_sensors/run_usd_camera.py

```
Program aborted due to an unhandled Error:
Unable to find target for this triple (no targets are registered)Aborted (core dumped)
```
[log.txt](https://github.com/NVIDIA-Omniverse/orbit/files/14772027/log.txt)

### System Info

Describe the characteristic of your environment:

- Commit: main branch
- Isaac Sim Version: 2023.1.1-rc.8+2023.1.688.573e0291.tc(orbit)
- OS: Ubuntu 20.04.6
- GPU: NVIDIA GeForce RTX 2080 Ti (sm_75)
- CUDA: 11.5
- GPU Driver: 525.85.05

### Additional context

So far all other tutorials works well, this error appear also in other tutorials when running in cpu mode. I am in remote server with no physical display connected to the desktop computer.
### Checklist

- [ x] I have checked that there is no similar issue in the repo (**required**)
- [ x] I have checked that the issue is not in running Isaac Sim itself and is related to the repo

### Acceptance Criteria

Add the criteria for which this task is considered **done**. If not known at issue creation time, you can add this once the issue is assigned.

- [ ] Can use the camera sensor for RL environment

**Comment by Mayankm96** [comment]
I think this is because of the reason here:

> I am in remote server with no physical display connected to the desktop computer.

Does it work when you run headless?

**Comment by gonzalezJohnas** [comment]
Everything works in headless mode, except for the sensor camera that's is why it is weird. I can successfully run other examples with this setup.

**Comment by Mayankm96** [comment]
For running camera headless, you probably need to pass `--offscreen_render` flag as well.

**Comment by gonzalezJohnas** [comment]
To clarify my problem, all other tutorial code works ( I can see the IsaacSim window open), just when getting to the sensor and more particularly run_usd_camera.py I have a core dumped. I tried in headless mode also and same error, this happens only when I tried to use sensors.

**Comment by gonzalezJohnas** [comment]
After debugging the program crashed with the error:

>Program aborted due to an unhandled Error:
>Unable to find target for this triple (no targets are registered)Aborted (core dumped)

Only when using the CameraCfg object, other sensor works well (RayCaster, ContactSensor). The error appear either when calling **scene.reset()** or accessing **scene['camera']**.

In IsaacSim I can create a camera and see the rendering from the perspective view, it seems to be a problem at Orbit level.

**Comment by gonzalezJohnas** [comment]
Hello, sorry to stress again about this bug but do you have some update on why the Camera sensors produce this error? Should I pull a specific version ??

**Comment by YitianShi** [comment]
Hi I fixed somehow the same problem through:
```
sudo prime-select nvidia
```
if your 
```
sudo prime-select query
```
gives `on-demand`

**Comment by gonzalezJohnas** [comment]
Thank you for the tip, but still have the error following your suggestion.

**Comment by ninamortensen** [comment]
> Thank you for the tip, but still have the error following your suggestion.

Hi, I have the same issue, have you by any chance resolved this?

**Comment by gonzalezJohnas** [comment]
For me plugging a monitor resolve the issue, I am still using ssh to work but that was the only solution to make the images works.
